_board/trinket_m0_haxpress.md

```diff
@@ -4,8 +4,7 @@ board_id: "trinket_m0_haxpress"
 title: "Trinket M0 Haxpress Download"
 name: "Trinket M0 Haxpress"
 manufacturer: "Dave Astels"
-board_url:
- - "http://daveastels.com/trinket-m0-express-hack.html"
+board_url: https://web.archive.org/web/20210727135908/http://daveastels.com/trinket-m0-express-hack.html
```

Comment:
I think it's best to make formatting match all of the other device config files with the board URL on it's own line with a hyphen at the beginning.
```
 - "https://web.archive.org/web/20210727135908/http://daveastels.com/trinket-m0-express-hack.html"
```

I'm not entirely certain if that syntax is 100% necessary, and I also don't know for sure where this board_url value gets used, but I think keeping it consistent across devices whenever possible is best.